feat: built ui runs on gotham

fede opened the pull request.

BlackMcDevil commented on README.md:
````diff
@@ -30,6 +30,8 @@ yarn install
 yarn start
 ```
 
+You can read more in the [UI README](./src/ui/README.md).
````
Didn't you delete the Readme file in #8 ?

fede replied:
Yes, this is a rebase issue, will remove this line again

fede replied:
Fixed, BTW I added contributors field on the package.json, add yourself! @BlackMcDevil

sireliah commented:
When running cargo run, we're supposed to get all static deps from /dist/, right? In the browser I'm getting:


```
404 GET | http://localhost:3000/web_modules/react.js
404 GET | http://localhost:3000/web_modules/react-dom.js
```


We should provide those libs statically I guess?

fede commented:
> When running cargo run, we're supposed to get all static deps from /dist/, right? In the browser I'm getting:
> 
> ```
> 404 GET | http://localhost:3000/web_modules/react.js
> 404 GET | http://localhost:3000/web_modules/react-dom.js
> ```
> 
> We should provide those libs statically I guess?
> 
> Edit: alternatively some README description how to run this would be helpful.

Found the problem! let me fix it on this PR

fede commented:
> When running cargo run, we're supposed to get all static deps from /dist/, right? In the browser I'm getting:
> 
> ```
> 404 GET | http://localhost:3000/web_modules/react.js
> 404 GET | http://localhost:3000/web_modules/react-dom.js
> ```
> 
> We should provide those libs statically I guess?
> 
> Edit: alternatively some README description how to run this would be helpful.

Fixed and added some more info the root README
